src/Commands/Multidev/CreateCommand.php

```diff
@@ -17,6 +17,7 @@ class CreateCommand extends TerminusCommand implements SiteAwareInterface
      * @authorized
      *
      * @name multidev:create
+     * @alias env:create
```

Comment:
Also I have been using the `@command` annotation. Is there a difference or preference?